src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -154,6 +191,16 @@ static bool _updateTransform(LottieTransform* transform, float frameNo, bool aut
         _rotateX(&matrix, transform->rotationEx->x(frameNo, exps));
     }
 
+    auto skewAngle = transform->skewAngle(frameNo, exps);
```

Comment:
In most cases, `skewAngle` and `skewAxis` will have 0 values (default). We can probably skip these defaults quickly.

Reply:
yes, this check (and more) are done inside ```_skew```:
```if (mathZero(fmod(angleDeg, 360.0f))) return;```
I can remove fmod...360 if it's wasting of time

Reply:
maybe something like this
```
if (skewAngle != 0.0f) {
   //apply the skew transform
    auto skewAxis = transform->skewAxis(frameNo, exps);
    _skew(&matrix, skewAngle, skewAxis);
}
```